internal/backend/akmodules/os/os.go

```diff
@@ -0,0 +1,98 @@
+package os
```

Comment:
A general comment. The fact we need both this one and `os` in `integrations` shows a design issue.

Reply:
mistake.